### PR TITLE
WIP: add Pickling of Data

### PR DESCRIPTION
**Pull request by jcarpent**

### CODE REVIEW

File: src/multibody/joint/joint-free-flyer.hpp [hunk 1]
```diff
@@ -192,7 +192,7 @@ namespace pinocchio
     : M(1)
     , U(U_t::Zero())
     , Dinv(D_t::Zero())
-    , UDinv(UD_t::Zero())
+    , UDinv(UD_t::Identity())
```
Comment (gabrielebndn) [inline src/multibody/joint/joint-free-flyer.hpp]: For consistency, shouldn't it be UDinv = U * Dinv ?

Comment (jcarpent) [reply src/multibody/joint/joint-free-flyer.hpp]: When you do the math, this quantity is always equal to Identity ;)

Comment (gabrielebndn) [reply src/multibody/joint/joint-free-flyer.hpp]: I understand it and I do agree with this change. What I mean is, shouldn't you change U and Dinv too, so to be consistent?

Comment (jcarpent) [reply src/multibody/joint/joint-free-flyer.hpp]: U and UDInv depends on the Inertia I input ;)

Comment (gabrielebndn) [reply src/multibody/joint/joint-free-flyer.hpp]: Oh, I see. Well, as they say, sorry for the noise

File: src/multibody/joint/joint-revolute.hpp [hunk 2]
```diff
@@ -120,16 +120,16 @@ namespace pinocchio
...
           res.rotation().col(2).noalias() = m_cos * m.rotation().col(2) + m_sin * m.rotation().col(0);
+          res.rotation().col(1) = m.rotation().col(1);
           res.rotation().col(0).noalias() = res.rotation().col(1).cross(res.rotation().col(2));
           break;
         }
         case 2:
         {
-          res.rotation().col(2) = m.rotation().col(2);
           res.rotation().col(0).noalias() = m_cos * m.rotation().col(0) + m_sin * m.rotation().col(1);
           res.rotation().col(1).noalias() = res.rotation().col(2).cross(res.rotation().col(0));
```
Comment (jmirabel) [inline src/multibody/joint/joint-revolute.hpp]: It seems to me that `res.rotation().col(2)` is not initialized, is it ?

**Pull request merged**